routes/index.py

```diff
@@ -1,9 +1,12 @@
 from flask import render_template, Blueprint
-from globals.globals import app
 
 index_blueprint = Blueprint('index', __name__)
-
+home_blueprint = Blueprint('home',__name__)
 
 @index_blueprint.route("/")
 def index():
+    return render_template("index.html")
+
+@home_blueprint.route("/home")
+def home():
```

Comment:
What's the difference between index and home? This looks like duplicate code with a different name.